src/index.js

```diff
@@ -77,8 +77,13 @@ const completeTodo = (element) => {
   element.classList.toggle(checked);
   element.classList.toggle(unchecked);
   element.parentNode.querySelector('.text').classList.toggle(lineThrough);
-
-  LIST[element.id].completed = !LIST[element.id].completed;
+  if (element.classList.contains('fa-check-square')) {
+    LIST[element.id - 1].completed = true;
+  } else if (element.classList.contains('fa-square-o')) {
+    LIST[element.id - 1].completed = false;
+  }
+  // const li = LIST[element.id].completed = LIST[element.id].completed+1 ? false : true;
+  // console.log(li)
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.